packages/clay-pagination-bar/src/PaginationBarWithBasicItems.tsx

```diff
@@ -127,6 +127,13 @@ export const ClayPaginationBarWithBasicItems: React.FunctionComponent<IProps> =
 	totalItems,
 	...otherProps
 }: IProps) => {
+	let showDropDown = true;
+
+	if (!deltas) {
+		deltas = defaultDeltas;
```

Comment:
I think `defaultDeltas` are unnecessary in this case.

thinking about the usage:

* When passing `deltas`: 

`defaultDeltas` will be ignored and the given `deltas` will be used ✅ 

* When not passing `deltas`:

`defaultDeltas` value will be used in this case, to not introduce a breaking change ❌ 

* When using a "falsy" value to `deltas` :

`defaultDeltas` still be used. ❌ 

Would you mind creating another flag like `disableItemsPerPage` or `disableDeltas` and just use it instead of `showDropDown`? What do you think?

Reply:
Hey @diegonvs 

> I think `defaultDeltas` is unnecessary in this case.

I don't understand what you mean, because it's already used (i.e. before my changes) if the `deltas` prop is [not passed](https://github.com/liferay/clay/blob/d8251c47262ccff9e715baf050c2713b075b11fa/packages/clay-pagination-bar/src/PaginationBarWithBasicItems.tsx#L119) and if the `activeDelta` prop isn't passed it gets it's value from there.

> When using a "falsy" value to deltas

I don't understand what you mean by a "falsy" value since `deltas`, see [here](https://codesandbox.io/s/determined-mendeleev-mowsd) 

> Would you mind creating another flag like disableItemsPerPage or disableDeltas and just use it instead of showDropDown? What do you think?

I don't mind adding another prop, but it would be nice to make it explicit: `disableDeltas` sounds like something that would be used to "ignore the deltas that are passed", but it's not the case. The only thing we want to do is "hide" the DropDown, so I'd make it as explicit and simple as possible. 

Actually, several people asked me what this "delta" prop means because they don't understand the meaning and it's already creating confusion.